test/jetstream_test.go

```diff
@@ -2738,7 +2815,9 @@ func TestJetStreamRedeliverCount(t *testing.T) {
 				if err != nil {
 					t.Fatalf("Unexpected error: %v", err)
 				}
-				sseq, dseq, dcount := o.ReplyInfo(m.Reply)
+
+				sseq, dseq, dcount, _ := o.ReplyInfo(m.Reply)
```

Comment:
We will need to make this cleaner in the actual clients.

Reply:
jsm has a helper that parse and returns a little object that gives access to all the properties, so no existing code need changing

Reply:
ok will coordinate as we start adding to nats.go